### PR TITLE
Add another state listener to the Slack notification

### PR DESCRIPTION
**Pull request by chiaberry**
We have some flows failing, but the failures aren't being reported to Slack. I want to try adding another state to listen for to see if it will show up in slack. 

https://docs.prefect.io/api/latest/engine/state.html#triggerfailed

### CODE REVIEW

File: flows/MDS/bird.py [hunk 1]
```diff
@@ -29,9 +29,8 @@
 current_environment = os.getenv("PREFECT_CURRENT_ENVIRONMENT", "staging")
 
 # Set up slack fail handler
-handler = slack_notifier(only_states=[Failed])
+handler = slack_notifier(only_states=[Failed, TriggerFailed])
```
Comment (chiaberry) [inline flows/MDS/bird.py:32]: I would like to try to see if this state happens when the flows don't start running at all.

Comment (Charlie-Henry) [reply flows/MDS/bird.py:32]: [It looks like](https://docs.prefect.io/api/latest/engine/state.html#state-2) `TriggerFailed` is a substate of the `Failed` state. Possible there's a bug and this isn't working how they thought?
```
my_state = TriggerFailed()
my_state.is_failed() # returns True
```

The silent error in the task from last night is actually technically in a `pending` state?
Screen Shot 2022-02-18 at 3 43 22 PM

**Review by mateoclarke — APPROVED** [review]
I'm good with this but seeing Charlie's comment makes me wonder if it would have any effect.

**Comment by Charlie-Henry** [comment]
I guess silent failures haven't been an issue for us now that we aren't getting these failures anymore but I saw this [issue](https://github.com/PrefectHQ/prefect/issues/5652) mentioned in the Prefect slack about this exact thing. 

[Slack thread](https://prefect-community.slack.com/archives/CL09KU1K7/p1649265574670459)

**Pull request closed without merging**